x-pack/plugins/security_solution/public/cases/components/create/form_context.tsx

```diff
@@ -72,6 +74,10 @@ export const FormContext: React.FC<Props> = ({
           settings: { syncAlerts },
         });
 
+        if (afterCaseCreated && updatedCase) {
```

Comment:
Could we do the attaching of an alert in the `onSuccess` callback?

Reply:
It was that before but we noticed that we need to first attach the alert to the case and then push the case to the external service.

Reply:
Gotcha. Could both the push and the attaching of alerts be done in the `onSuccess`? I guess we'd still need some if logic in there since we don't always attach alerts when a case is created?

Reply:
As we discussed, the pushing should remain at `form_context` and we cannot do the attaching to the alert in the `onSuccess` callback?